chips/src/main/java/com/decathlon/vitamin/compose/chips/NoRippleInteractionSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.decathlon.vitamin.compose.chips
+
+import androidx.compose.foundation.interaction.Interaction
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+
+internal class NoRippleInteractionSource : MutableInteractionSource {
```

Comment:
Because `Chip` component from Compose UI has `enabled` parameter, this is really necessary? 🤔

Reply:
To move to `foundation`